cmake/Toolchain-mingw.cmake

```diff
@@ -20,40 +19,3 @@ if(NOT EXISTS "${CMAKE_SOURCE_DIR}/dependencies/mingw-xaudio/include")
     endif()
 endif()
 
-# hack for ninja in msys2
-if(WIN32 AND CMAKE_GENERATOR STREQUAL Ninja AND NOT "$ENV{MSYSTEM_PREFIX}" STREQUAL "")
-    set(MSYS ON)
-endif()
-
-if(MSYS AND CMAKE_CXX_COMPILER_ID STREQUAL Clang)
-    if($ENV{MSYSTEM} STREQUAL CLANG64)
-        cygpath(prefix "$ENV{MSYSTEM_PREFIX}/x86_64-w64-mingw32")
-        list(APPEND CMAKE_PREFIX_PATH "${prefix}")
-    elseif($ENV{MSYSTEM} STREQUAL CLANG32)
-        cygpath(prefix "$ENV{MSYSTEM_PREFIX}/i686-w64-mingw32")
-        list(APPEND CMAKE_PREFIX_PATH "${prefix}")
-    endif()
-
-    set(CMAKE_PREFIX_PATH "${CMAKE_PREFIX_PATH}" CACHE INTERNAL "prefix search path for find_XXXX" FORCE)
-endif()
-
-# link libgcc/libstdc++ statically on mingw
-# and adjust link command when making a static binary
-if(CMAKE_COMPILER_IS_GNUCXX AND VBAM_STATIC)
```

Comment:
This block has to stay. I need to revisit linking on UCRT64/MINGW64 and then I will make any necessary adjustments.

Reply:
I will probably make a separate static toolchain helper later on, because I'm also interested in making a "static" linux binary, and I'm fairly close to being able to do that.

Reply:
Reverted